recipes/libxlsxwriter/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "1.0.0":
+    url: "https://github.com/jmcnamara/libxlsxwriter/archive/RELEASE_1.0.0.tar.gz"
+    sha256: "8b353379333c323d14a9d265cd2491d3a6c0032c8d6ec2141f10b82ab66a087c"
+patches:
+  "1.0.0":
+    - base_path: "source_subfolder"
+      patch_file: "patches/conan.patch"
```

Comment:
if you would be so kind to rename the patch, something like "001-fix-zip-header-include.patch". This way when we add a new version it's easy to know what we needed to fix.